src/DotVVM.Framework/Resources/Scripts/DotVVM.Polyfills.ts

```diff
@@ -0,0 +1,10 @@
+﻿(function () {
+    if (typeof Promise === 'undefined' || !self.fetch) {
+        var resource = <HTMLScriptElement>document.createElement('script');
```

Comment:
The cast to `HTMLScriptElement` should not be required by TypeScript, it should know that form the `'script'` in parameter. Are you sure your TS version is up to date?

Reply:
You're right, the cast isn't required. I didn't know that.